org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java

```diff
@@ -450,80 +579,67 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     }
   }
 
-  /**
-   * Returns a directory-like listing of all the paths to resources which are located in other (dependent) JAR files.
-   * Those JAR files are placed in the web application's '/WEB-INF/lib' directory, and their resources located in the
-   * '/META-INF/resources' folder.
-   * <p>
-   * This method works for both, packed and unpacked JAR files, which is crucial when running the application from
-   * within the IDE. see 'javax.servlet.ServletContext.getResourcePaths(String)' for the specification.
-   */
-  protected static Set<String> getResourcePathsFromDependentJars(ClassLoader classloader, String path) {
-    path = path.endsWith(URIUtil.SLASH) ? path : path + URIUtil.SLASH;
+  protected static class P_WebAppContext extends WebAppContext {
```

Comment:
Extract to own class, removing all static methods in Application class?

Reply:
Extracted to class DevelopmentWebAppContext